app/api/attendees.py

```diff
@@ -47,13 +47,12 @@ def before_post(self, args, kwargs, data):
                 {'pointer': '/data/relationships/ticket'},
                 "Ticket belongs to a different Event"
             )
-
-        if db.session.query(TicketHolder.id).filter_by(
-                ticket_id=int(data['ticket']), event_id=int(data['event'])
-                ).scalar() is not None:
+        # Check if the ticket is already sold out or not.
+        if get_count(db.session.query(TicketHolder.id).
+                     filter_by(ticket_id=int(data['ticket']), deleted_at=None)) >= ticket.quantity:
             raise ConflictException(
                 {'pointer': '/data/attributes/ticket_id'},
-                "Attendee with this ticket already exists for the same event"
+                "Ticket already sold out"
```

Comment:
Could there be a better message? Like _all such_ tickets have been sold out?

Reply:
Does ticket id refer to just 1 ticket, or to the tickets in general for an event.

Reply:
Tickets in general. 
for eg. Ticket has a field quantity. So if the quantity is 100, then 100 people can buy that ticket.

Reply:
Or maybe just "Sold out"